.gitignore

```diff
@@ -46,3 +46,5 @@ nosetests.xml
 
 # Vim
 *.sw[op]
+
+.cache/
```

Comment:
The tests leave .cache folder behind, and I kept accidentally trying to commit it because I get lazy with git add -A